kivy/uix/bubble.py

```diff
@@ -80,7 +80,7 @@ class BubbleButton(Button):
     Rather use this BubbleButton widget that is already defined and provides a
     suitable background for you.
     '''
-    pass
+    always_release = BooleanProperty(True)
```

Comment:
@dessant  if the always_release is not explicitly made True for BubbleButton class (as highlighted above) then the on_release event won't be triggered by the button here https://github.com/kivy/kivy/blob/master/kivy/data/style.kv#L202-L217 making the bubbles non-functioning. Other way would be to use on_press instead of on_release in style.kv...